app/styles/_core.less

```diff
@@ -98,8 +98,15 @@
     font-style: normal;
     margin-top: -5px;
   }
+  .ui-select-match-text > span {
+      width: 85%;
+      overflow: hidden;
+      text-overflow: ellipsis;
+      position: absolute;
+  }
+
   .btn {
-    padding-right: 25px; // prevent text from showing behind select arrow
+    padding-right: 10px; // prevent text from showing behind select arrow
```

Comment:
How does that affect the existing selects in our UI?

I'd rather not change existing the styles and instead just have an empty option in the select.